package.json

```diff
@@ -17,7 +17,7 @@
   "bin": {
     "odata-openapi3": "lib/cli.js"
   },
-  "main": "lib/csdl2openapi.js",
+  "exports": "./lib/csdl2openapi.js",
```

Comment:
Why does the package no longer have an entry point `main`?

Reply:
https://nodejs.org/api/packages.html#package-entry-points

> For new packages targeting the currently supported versions of Node.js, the ["exports"](https://nodejs.org/api/packages.html#exports) field is recommended. For packages supporting Node.js 10 and below, the ["main"](https://nodejs.org/api/packages.html#main) field is required. If both ["exports"](https://nodejs.org/api/packages.html#exports) and ["main"](https://nodejs.org/api/packages.html#main) are defined, the ["exports"](https://nodejs.org/api/packages.html#exports) field takes precedence over ["main"](https://nodejs.org/api/packages.html#main) in supported versions of Node.js.

We require at minimum Node.js **14** and now follow the recommendation for `exports`, which makes `main` superfluous.